osbenchmark/builder/downloaders/opensearch_distribution_downloader.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os.path
+
+from osbenchmark.builder.downloaders.downloader import Downloader
+from osbenchmark.builder.downloaders.repositories.opensearch_distribution_repository_provider import \
+    OpenSearchDistributionRepositoryProvider
+from osbenchmark.builder.utils.path_manager import PathManager
+from osbenchmark.exceptions import ExecutorError
+
+
+class OpenSearchDistributionDownloader(Downloader):
```

Comment:
Do we only support OpenSearch?

Reply:
For now, yes. Supporting ES is a non-MVP "nice to have"